### ISSUE TITLE
Resolve "X509v3 Subject Alternative Name: critical"

### ISSUE THREAD

**Issue by berendt** [osism/testbed#1390]
Source: https://github.com/osism/testbed/issues/1282#issuecomment-1308622772

Ok, since this issue is still open, I'll re-use it to request an update for the certificate, because Keycloak rejects it (with a java exception) as non-RFC compliant:

```
Caused by: java.security.cert.CertificateParsingException: X.509 Certificate is incomplete: SubjectAlternativeName extension MUST be marked critical when subject field is empty
```

I guess the error message is referring to last sentence of the paragraph of https://www.rfc-editor.org/rfc/rfc5280#section-4.1.2.6

This issue is blocking us as it occurs when you want to do federation between SCS Keycloak instandes (e.g. also simply federating to a separate local Keycloak realm).

@JuanPTM did an experiment and manually created a new certificate locally in a testbed deployment so that
```
openssl x509 -text -in /opt/traefik/certificates/default.cert -noout | grep -C 2 -i alt
```
shows `X509v3 Subject Alternative Name: critical`. After deploying that and restarting traefik, the problem seems to have been solved. It would be great to also fix it in the default testbed deployments.

Something like this had to be added to the openssl config:
```
[SAN]
subjectAltName=critical
```

**Comment by berendt** [comment]
The certificate for Keycloak looks good. Looks like only ./environments/kolla/certificates/ca/testbed.crt is not correct. 

```
❯ openssl x509 -text -in ./environments/custom/files/keycloak/cert.crt -noout | grep -C 2 -i alt
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Subject Alternative Name:
                DNS:*.testbed.osism.xyz
            X509v3 Subject Key Identifier:
```

**Comment by osfrickler** [comment]
@reqa new certificates are available in https://github.com/osism/testbed/pull/1394 , can you please confirm that these work as expected?

**Comment by reqa** [comment]
The change didn't take any effect in a fresh testbed newly deployed today by @fkr:

![image](https://user-images.githubusercontent.com/8189985/202717830-719e2819-0069-4fde-a6c3-03092eaab130.png)

This should read `X509v3 Subject Alternative Name: critical`. I'll try to find out where this got stuck.

**Comment by fkr** [comment]
@reqa I did my initial deployment yesterday afternoon (when the change was not merged yet).

**Comment by reqa** [comment]
The problem is, that `create_wildcard.yml` would need the change, but instead `create_ca.yml` and `create_manager.yml` have been changed. I'm pretty sure that `create_ca.yml` doesn't need the change at all, as the CA-Cert already has the critical bits set by default.

I attempted to simply make the change locally in the current testbed and running `ansible-playbook create_wildcard.yml` but that fails because I don't seem to have access to the CA key (the playbook complains about a missing `testbed-ca-certificate.key`). If somebody has any hint where I can retrieve that key, then I could help myself.
